index.html

```diff
@@ -35,6 +35,152 @@ <h2>I’am a Software Developer</h2>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button class="card-btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <img src="assets/images/Card Placeholder.png" alt="text book image with a dark mask over it">
+                        <div class="right-block">
+                            <h3>Profesional Art Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button class="card-btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <img src="assets/images/Card Placeholder.png" alt="text book image with a dark mask over it">
+                        <div class="right-block">
+                            <h3>Profesional Art Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button class="card-btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <img src="assets/images/Card Placeholder.png" alt="text book image with a dark mask over it">
+                        <div class="right-block">
+                            <h3>Profesional Art Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button class="card-btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="about">
```

Comment:
In order to improve your code consider using a `section` tag here. Using a `section` tag instead of a `div` can improve the accessibility and search engine optimization of a web page 🌐 , as it gives the content a more meaningful structure and context ℹ️ . Additionally, it makes the code more readable, making it easier to understand the structure of the page and how the different elements are related to one another. 🤓